data/actions/scripts/other/change_gold.lua

```diff
@@ -6,7 +6,7 @@ local config = {
 
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	local coin = config[item:getId()]
-	if coin.changeTo and item.type == 100 then
+	if coin.changeTo and item:getSubType() == 100 then
```

Comment:
shouldn't here be getCount instead? 🤔

Reply:
I followed how [combat](https://github.com/otland/forgottenserver/blob/master/data/lib/compat/compat.lua#L76-L77) works, I also tested the script and it works normally.

Reply:
But I will change it to `getCount()` . it is explicit